x-pack/plugins/security_solution/public/common/components/with_hover_actions/index.tsx

```diff
@@ -64,10 +66,16 @@ export const WithHoverActions = React.memo<Props>(
 
     const content = useMemo(() => <>{render(showHoverContent)}</>, [render, showHoverContent]);
 
-    const isOpen = hoverContent != null && (showHoverContent || alwaysShow);
+    useEffect(() => {
+      setIsOpen(hoverContent != null && (showHoverContent || alwaysShow));
+    }, [hoverContent, showHoverContent, alwaysShow]);
 
-    const popover = useMemo(() => {
-      return (
+    useEffect(() => {
+      setShowHoverContent(false);
+    }, [closePopOverTrigger]);
```

Comment:
consider declaring `setShowHoverContent` as a dependency

Reply:
there is no need because it is from useState

Reply:
here the explanation -> https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often